icons/tradingview.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>TradingView</title><path d="M9 6H0v4.5h4.5V18H9V6zm10.5 12h-6L18 6h6l-4.5 12zm-6.75-7.5c1.2426 0 2.25-1.0074 2.25-2.25S13.9926 6 12.75 6 10.5 7.0074 10.5 8.25s1.0074 2.25 2.25 2.25z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>TradingView</title><path d="M9.465 18.085H4.732v-7.437H0V5.915h9.465v12.17zm9.465 0h-5.41l5.07-12.17H24l-5.07 12.17zm-5.41-12.17a2.704 2.704 0 1 1 0 5.409 2.704 2.704 0 0 1 0-5.409z"/></svg>
```
This should fix the differences. Perplexed as to why the previous suggestion didn't work